pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -24,6 +24,8 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
ughhh we need to get rid of this dependency

Reply:
> grep -r machinery pkg/ | grep sets
> pkg/build/nodeimage/buildcontext.go:    "k8s.io/apimachinery/pkg/util/sets"
> pkg/cluster/internal/kubeconfig/internal/kubeconfig/paths.go:   "k8s.io/apimachinery/pkg/util/sets"
> pkg/cluster/internal/providers/docker/provider.go:      "k8s.io/apimachinery/pkg/util/sets"
> pkg/cluster/internal/providers/podman/provider.go:      "k8s.io/apimachinery/pkg/util/sets"
> pkg/cluster/internal/providers/common/images_test.go:   "k8s.io/apimachinery/pkg/util/sets"
> pkg/cluster/internal/providers/common/images.go:        "k8s.io/apimachinery/pkg/util/sets"
> pkg/cmd/kind/load/docker-image/docker-image.go: "k8s.io/apimachinery/pkg/util/sets"

Reply:
pkg/internal/sets after #2539

Reply:
(please group internal imports on their own, at the bottom, I had hoped to avoid internal imports in cmd/ but I think we can make an exception here, people re-implementing the cmd/ logic using the kind public APIs can just import k8s.io/apimachinery sets themselves if they need to implement this bit of logic)